pkg/ovs/openflow/interfaces.go

```diff
@@ -156,8 +150,19 @@ type FlowBuilder interface {
 	MatchARPSpa(ip net.IP) FlowBuilder
 	MatchARPTpa(ip net.IP) FlowBuilder
 	MatchARPOp(op uint16) FlowBuilder
-	MatchCTState(value string) FlowBuilder
-	MatchCTMark(value string) FlowBuilder
+	MatchCTStateNew() FlowBuilder
+	MatchCTStateUnNew() FlowBuilder
+	MatchCTStateRel() FlowBuilder
+	MatchCTStateUnRel() FlowBuilder
+	MatchCTStateRpl() FlowBuilder
+	MatchCTStateUnRpl() FlowBuilder
+	MatchCTStateEst() FlowBuilder
+	MatchCTStateUnEst() FlowBuilder
+	MatchCTStateTrk() FlowBuilder
+	MatchCTStateUnTrk() FlowBuilder
+	MatchCTStateInv() FlowBuilder
+	MatchCTStateUnInv() FlowBuilder
```

Comment:
How about `MatchCTStateNew(set bool)` to save half of methods?

Reply:
Done.